src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -138,6 +161,84 @@ Dictionary
               </label>
             </div>
             <br />
+
+
+            <div id="other-search">
+              <div className="form-check">
+                <input
+                  className="form-check-input"
+                  type="radio"
+                  name="exampleRadios"
```

Comment:
Did you intentionally call it exampleRadios?

Reply:
no, it is now fixed.